backend/src/modules/Announcements/service/Announcement.service.spec.js

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
Ten plik do śmietnika?
Zasada YAGNI
(ale że deadline i może niezbędne zaraz, to daję approve (jeśli nie to usuń proszę)
https://pl.wikipedia.org/wiki/YAGNI